packages/pretty_expressive/pretty_expressive.0.5/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "A pretty expressive printer"
+description:
+  "A pretty printer implementation of 'A Pretty Expressive Printer' (OOPSLA'23), with an emphasis on expressiveness and optimality."
+maintainer: ["Sorawee Porncharoenwase <sorawee.pwase@gmail.com>"]
+authors: ["Sorawee Porncharoenwase <sorawee.pwase@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/sorawee/pretty-expressive-ocaml"
+doc: "https://sorawee.github.io/pretty-expressive-ocaml/"
+bug-reports: "https://github.com/sorawee/pretty-expressive-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "3.10"}
+  "mdx" {>= "2.3.0" & <= "2.3.1" & with-test}
```

Comment:
Is this still needed?

Reply:
#25476 is just merged. So I guess `(& (>= 2.3.0) (<> 2.4.0))` could also be used. Is it worth making the change?

Reply:
I made the aforementioned change